Add Summer 2021 T&F Fanart Contest Results newspost (#6005)
ppy/osu-wiki: news/2021-08-23-summer-tf-fanart-contest-results.md

---
layout: post
title: "Summer 2021 Track and Field Fanart Contest Results"
date: 2021-08-23 12:00:00 +0000
---

> The race is over, now witness the podium finish in all its glory with 15 new Summer seasonal backgrounds!

Reviewer: ```suggestion
The race is over — now witness the podium finish in all its glory with 15 new Summer seasonal backgrounds!
```

Without dallying about any further, let's get straight to the results:

*Click any of the banners to be taken to a high-resolution version of the image!*

[![](https://assets.ppy.sh/contests/133/banners/Porukana.jpg)](https://assets.ppy.sh/contests/133/winners/Porukana.png)

Vaulting into first place with **2950** votes is fanart contest veteran [**Porukana**](https://osu.ppy.sh/users/12992775)'s take on pippi executing a flawless pole vault jump in an understatedly wonderful track outfit. Astute viewers will note something slightly amiss with her shoes — the laces seem to be untied!

Such antics can only come from Yuzu or one of his gadgets. Aside from that, there is **absolutely nothing else** amiss. Absolutely nothing.

A big congratulations goes to Porukana, who has been a regular participant in these contests since Summer 2019 and now claims the top spot all for themselves! We adore our regulars and especially love watching many of them chart a steady rise up through the voting totals as they improve year by year.

[![](https://assets.ppy.sh/contests/133/banners/Ravioli-.jpg)](https://assets.ppy.sh/contests/133/winners/Ravioli-.png)

> **2586** voters could not help but listen to [**Ravioli-**](https://osu.ppy.sh/users/8678404)'s take on Mocha pleading with pippi and Chirou to slow down a bit mid way through a marathon run.

Reviewer: ```suggestion
**2586** voters could not help but listen to [**Ravioli-**](https://osu.ppy.sh/users/8678404)'s take on Mocha pleading with pippi and Chirou to slow down a bit midway through a marathon run.
```

It's a good thing Chirou is staying hydrated in this one, but it is a terrible shame that the cup she's drinking from is occluding her runner's number. I guess we'll never know what it was.

[![](https://assets.ppy.sh/contests/133/banners/SvenJing.jpg)](https://assets.ppy.sh/contests/133/winners/SvenJing.png)

> A crowd of **2555** voters cheered on [**SvenJing**](https://osu.ppy.sh/users/11215996)'s most acrobatic pippi vaulting over a bar within a fully packed stadium. We'd like to believe this is what the World Cups will look like some day!

Reviewer: ```suggestion
A crowd of **2555** voters cheered on [**SvenJing**](https://osu.ppy.sh/users/11215996)'s most acrobatic pippi vaulting over a bar in a fully packed stadium. We'd like to believe this is what the World Cups will look like some day!
```

[![](https://assets.ppy.sh/contests/133/banners/Dreamxiety.jpg)](https://assets.ppy.sh/contests/133/winners/Dreamxiety.png)

Contest regular [**Dreamxiety**](https://osu.ppy.sh/users/13103233) is back at it again with their take on pippi finishing a marathon to the applause of **2251** voters with an especially interesting layered hitcircle top.

[![](https://assets.ppy.sh/contests/133/banners/Ayamy.jpg)](https://assets.ppy.sh/contests/133/winners/Ayamy.png)

> **2068** voters joined [**Ayamy**](https://osu.ppy.sh/users/10275467) in reminiscing about pippi's track and field days with a charming photograph of a beautiful summer's day.

Reviewer: ```suggestion
**2068** voters joined [**Ayamy**](https://osu.ppy.sh/users/10275467) in reminiscing about pippi's track and field days with a charming photograph of a beautiful summer day.
```

It's a terrible shame that the airplane in the background is too blurry to have the number on its side visible. I imagine that would be a real bombshell otherwise.

[![](https://assets.ppy.sh/contests/133/banners/Kondroel.jpg)](https://assets.ppy.sh/contests/133/winners/Kondroel.png)

[**Kondroel**](https://osu.ppy.sh/users/14634525) returns once again with their signature style, depicting an insanely fashionable pippi preparing for a day's worth of events with most of the mascot crew (including a very squishy-cheeked Chirou and a curious Taikonator) during a cinematic sunrise, scoring **1922** well earned votes.

So excellently done!

[![](https://assets.ppy.sh/contests/133/banners/Kwms024.jpg)](https://assets.ppy.sh/contests/133/winners/Kwms024.png)

> Athletic energy is front and center in [**Kwms024**](https://osu.ppy.sh/users/11516014)'s entry, scoring **1760** votes from hyped up observers, showcasing the prowess of all the official mascots across every game mode. And a fish.

Reviewer: ```suggestion
Athletic energy is front and center in [**Kwms024**](https://osu.ppy.sh/users/11516014)'s entry, scoring **1760** votes from hyped-up observers, showcasing the prowess of all the official mascots across every game mode. And a fish.
```

There will be no talk of competitor numbers. None whatsoever.

[![](https://assets.ppy.sh/contests/133/banners/not%20slepp.jpg)](https://assets.ppy.sh/contests/133/winners/not%20slepp.png)

> **1657** voters watched in on [**not slepp**](https://osu.ppy.sh/users/14406940)'s rendition of Mocha and Alisa enjoying a bit of locker room chat. Poor Mocha looks like she's struggling, but then again, when isn't she? 

Reviewer: ```suggestion
**1657** voters watched in on [**not slepp**](https://osu.ppy.sh/users/14406940)'s rendition of Mocha and Alisa enjoying a bit of a locker room chat. Poor Mocha looks like she's struggling, but then again, when isn't she? 
```

[![](https://assets.ppy.sh/contests/133/banners/Dem4eg-.jpg)](https://assets.ppy.sh/contests/133/winners/Dem4eg-.png)

**1629** voters cheered on [**Dem4eg-**](https://osu.ppy.sh/users/7206818)'s pippi and Mocha to the finish. Did you see something especially big in this entry? Us too! Mocha's star is *huge*.

[![](https://assets.ppy.sh/contests/133/banners/McFriedFries.jpg)](https://assets.ppy.sh/contests/133/winners/McFriedFries.png)

> **1547** voters were wowed by the sheer cool factor of [**McFriedFries**](https://osu.ppy.sh/users/12694139)'s pippi mid jump. Just look at her! That's a lady who knows where she's going: right over that bar, and she's confident enough to give the crowd a nod while she's at it.

Reviewer: ```suggestion
**1547** voters were wowed by the sheer cool-factor of [**McFriedFries**](https://osu.ppy.sh/users/12694139)'s pippi mid-jump. Just look at her! That's a lady who knows where she's going: right over that bar, and she's confident enough to give the crowd a nod while she's at it.
```

[![](https://assets.ppy.sh/contests/133/banners/Phoebe%20Yu.jpg)](https://assets.ppy.sh/contests/133/winners/Phoebe%20Yu.png)

**1507** linked in to [**Phoebe Yu**](https://osu.ppy.sh/users/3607337)'s very creative take on a technologically enhanced and (presumably) housebound pippi plugging herself in to *track and field shenanigans*.

Bonus imaginary points for taking a very unexpected twist on the theme and setting!

[![](https://assets.ppy.sh/contests/133/banners/kabandayo.jpg)](https://assets.ppy.sh/contests/133/winners/kabandayo.png)

> [**kabandayo**](https://osu.ppy.sh/users/21567829) wowed over **1472** voters with their take on a baton pass moment between pippi and Mocha mid way through a race. Just look at how excited pippi is!

Reviewer: ```suggestion
[**kabandayo**](https://osu.ppy.sh/users/21567829) wowed over **1472** voters with their take on a baton pass moment between pippi and Mocha midway through a race. Just look at how excited pippi is!
```

**1222** voters joined Chirou, a don and the Taikonator in cheering on [**Tin Can**](https://osu.ppy.sh/users/10682807)'s pippi and Mocha through a baton marathon. Unfortunately for Mocha, it seems like she's about to be introduced to the ground or drop her baton — possibly both at the same time. Such is the struggle of an athlete.

[![](https://assets.ppy.sh/contests/133/banners/Sutsuka.jpg)](https://assets.ppy.sh/contests/133/winners/Sutsuka.png)

In an EXTREMELY unexpected twist, [**Sutsuka**](https://osu.ppy.sh/users/29089) left **1200** voters awestruck with their incredible take on osu! themed ancient Greek terracotta art, including a blazing prize amphora sporting all kinds of osu! game mode inspired motifs.

If you're at all familiar with this kind of historical art, you'll appreciate the incredible attention to detail that went into this piece, all while embracing the mascots you know and love.

A truly spectacular piece and easily one of the most memorable fanart entries I think I have ever seen thus far. Wow.

[![](https://assets.ppy.sh/contests/133/banners/Loichuu.jpg)](https://assets.ppy.sh/contests/133/winners/Loichuu.png)

> **1159** voters embraced the chaos (and cute) of [**Loichuu**](https://osu.ppy.sh/users/460210)'s most athletic scene depicting pippi undertaking a victory lap while Chirou drops her ice water bowl in an ill-fated pursuit. And of course, where there is Yuzu, there is *shenanigans*, as that rather ominously aimed disc might suggest...

Reviewer: ```suggestion
**1159** voters embraced the chaos (and cuteness) of [**Loichuu**](https://osu.ppy.sh/users/460210)'s most athletic scene depicting pippi undertaking a victory lap while Chirou drops her ice water bowl in an ill-fated pursuit. And of course, where there is Yuzu, there is *shenanigans*, as that rather ominously aimed disc might suggest...
```

---

[Download a zip file containing all the winning entries in high definition here!](https://assets.ppy.sh/contests/133/winners/Summer2021TFWinners.zip)

The [Summer 2021 Track and Field Fanart Contest listing](https://osu.ppy.sh/community/contests/133) awaits with the full rundown for everyone who entered, so make sure to check that all out and see where your favourite entries ended up in the tally if they didn't make it here.

> You'll find the finalists listed above are available in game now as the Summer 2021 seasonal backgrounds (and in osu!lazer too!). The top three artists have also earned themselves a special profile badge featuring their winning work, and everybody here also goes home with 3 months of osu!supporter to celebrate their victory.

Reviewer: ```suggestion
You'll find the finalists listed above are available in-game now as the Summer 2021 seasonal backgrounds (and in osu!(lazer) too!). The top three artists have also earned themselves a special profile badge featuring their winning work, and everybody here also goes home with 3 months of osu!supporter to celebrate their victory.
```

> Thank you to everybody who entered and voted, and we'll see you in only a couple of weeks for another Fan Art contest, because as you all know by now, the end of the year is just chock full of them.

Reviewer: ```suggestion
Thank you to everybody who entered and voted, and we'll see you in only a couple of weeks for another Fan Art contest, because as you all know by now, the end of the year is just chock-full of them.
```
